nata/styles.css

```diff
@@ -0,0 +1,45 @@
+body {
```

Comment:
If you fix HTML markup I guess you will have different CSS rules here, so I won't check them now.